product/src/main/java/com/yas/product/exception/BadRequestException.java

```diff
@@ -1,7 +1,21 @@
 package com.yas.product.exception;
 
+import com.yas.product.utils.MessagesUtils;
+
 public class BadRequestException extends RuntimeException {
-    public BadRequestException(final String message) {
-        super(message);
+
+    private String message;
```

Comment:
I think we should expose the error code. We might need to use them to compare them when handle exception